theano/sandbox/cuda/opt.py

```diff
@@ -1353,7 +1353,7 @@ def cast(x, dtype):
 
 @register_opt()
 @local_optimizer([tensor.nnet.CrossentropySoftmaxArgmax1HotWithBias])
```

Comment:
Thanks for this other fix. Just as for the ohter PR to don't break existing code, can you change this line by:

@local_optimizer([tensor.nnet.CrossentropySoftmaxArgmax1HotWithBias], 'local_gpu_crossentorpy_softmax_argmax_1hot_with_bias")

Idem for the other one bellow.